python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -20,17 +20,17 @@
 
 def set_partitions_hash(df, columns, npartitions):
     c = hash_object_dispatch(df[columns], index=False)
-    return np.mod(c, npartitions)
+    return cupy.mod(c, npartitions)
```

Comment:
These are actually fine because it goes through cupy via array function protocol: https://numpy.org/neps/nep-0018-array-function-protocol.html

Reply:
I had to change it to `cupy` as numpy doesn't seem to work, this was the error:
```python
>>> import cudf
>>> import numpy as np
>>> import cupy
>>> s = cudf.Series([1,2,3])
>>> np.mod(s, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: operand type(s) all returned NotImplemented from __array_ufunc__(<ufunc 'remainder'>, '__call__', 0    1
1    2
2    3
dtype: int64, 2): 'Series', 'int'
>>> cupy.mod(s, 2)
array([1, 0, 1])
>>> 
```

Reply:
@kkraus14, I have reverted the `cupy` change and instead implemented the missing universal functions in `Series`